app/scripts/modules/core/src/pipeline/config/triggers/helm/HelmTriggerExecutionStatus.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import { IExecutionTriggerStatusComponentProps } from 'core/domain';
+import { IHelmTrigger } from 'core/domain/IHelmTrigger';
+
+export const HelmTriggerExecutionStatus = (props: IExecutionTriggerStatusComponentProps) => {
+  const trigger = props.trigger as IHelmTrigger;
+  const name = trigger.artifactName;
+  const version = trigger.version;
+
+  return (
+    <li>
```

Comment:
Curious why this is a list element?

Reply:
The parent `ExecutionStatus` component renders the trigger-specific data under an `ul` element, and each `li` is a row in that section of the execution UI.

Reply:
I see, thanks! 👍